examples/models/llama/install_requirements.sh

```diff
@@ -17,6 +17,8 @@ pip install --no-use-pep517 "git+https://github.com/pytorch/ao.git@${TORCHAO_VER
 # Install tiktoken for tokenizer
 pip install lm_eval==0.4.2
```

Comment:
cc: @helunwencser This is problematic and it's already stopped working for for transformers since 4.45 release over a month ago.

Reply:
Resolved in #6555 by @helunwencser